pa_config/rca_idle_master.conf

```diff
@@ -1,5 +1,5 @@
 {
-  "analysis-graph-implementor": "com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.store.DummyGraph",
+  "analysis-graph-implementor": "com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.store.ElastiSearchAnalysisGraph",
```

Comment:
I think there is a typo here. is it supposed to be "ElasticSearchAnalysisGraph" ?

Reply:
Right, fixing this typo